cmd/antrea-controller/controller.go

```diff
@@ -130,6 +130,12 @@ func run(o *Options) error {
 	if err != nil {
 		return fmt.Errorf("error creating API server: %v", err)
 	}
+
+	err = apiserver.CleanupDeprecatedAPIServices(aggregatorClient)
```

Comment:
I didn't think of it too much but does the order matter here? should we remove deprecated APIServices before installing the new ones?

Reply:
Installation happens locally on controller, and new APIServices are already registered when applying the YAML. Before apiserver starts, K8s knows nothing about the new API groups. So, I think it does not matter.